<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64,64,64);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Dodaj 
rekord</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Zmień 
rekord</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Usuń 
rekord</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Wyjdź</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">background-color: 	rgb(192,192,192);
</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
